benchexec/tools/coveritest.py

```diff
@@ -0,0 +1,25 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.tools.cpachecker as cpachecker
+
+
+class Tool(cpachecker.Tool):
+    """Tool info for CoVeriTest."""
+
+    REQUIRED_PATHS = [
+        "lib/java/runtime",
+        "lib/*.jar",
+        "lib/native/x86_64-linux",
+        "scripts",
+        "cpachecker.jar",
+        "config",
+        "resources",
```

Comment:
As a CPAchecker developer, I am curious about where this path comes from? AFAIK there is nothing in current CPAchecker that would access such a directory.

Reply:
There is a new CoVeriTest version coming with an advanced time scheduler. The scheduler locates some resources in the resource folder. Changes to CPAchecker which would require the resources/ path were not pushed to trunk yet.